docs/security/non-events.md

```diff
@@ -0,0 +1,83 @@
+<!--[metadata]>
++++
+title = "Docker Security Non-events"
+description = "Review of security vulnerabilities Docker mitigated"
+keywords = ["Docker, Docker documentation,  security, security non-events"]
+[menu.main]
+parent = "smn_secure_docker"
+weight =-99
++++
+<![end-metadata]-->
+
+# Docker Security Non-events
+
+This page lists security vulnerabilities which Docker mitigated, such that
+processes run in Docker containers were never vulnerable to the bug—even before
+it was fixed. This excludes for the most part containers run as `--privileged`.
+
```

Comment:
I think I would say "This excludes containers run as `--privileged`, and in some cases containers run with the `SYS_ADMIN` capability."